guides/quickstart_guide.adoc

```diff
@@ -1,14 +1,19 @@
 = Getting Started
+
 :excerpt: Getting started is easy, lets find out how
 :title: Getting Started
 :order: 1
 :source-highlighter: highlightjs
 
 == Setup
 
-The SDK consists of 6 modules:
+The SDK consists of 3 main-modules:
 
-current-version = link:../version.txt[click]
+current-version =
```

Comment:
should this be empty?

Reply:
the version is included below, so it gets rendered from the version.txt file